### PR TITLE
changed time to 12 hour format

### PR DESCRIPTION
**Pull request by AjCodez**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 function update(){
     let time=new Date();
     let hrs=time.getHours();
+    suffix = hrs >= 12 ? "PM":"AM";
+    hrs=((hrs % 12) || 12);
     document.getElementById('hours').innerHTML=hrs;
```
Comment (Tusharsaxena3112) [inline script.js]: declare the variables always with something like let or var, this is JS, not python.

**Review by Tusharsaxena3112 — CHANGES_REQUESTED** [review]

**Review by AjCodez — COMMENTED** [review]
added var in front of variable names

**Comment by AjCodez** [comment]
I tried adding var or let in front of them, but time isn't showing on doing that.

**Review by Tusharsaxena3112 — APPROVED** [review]
Looks Good

**Pull request merged**